src/app/modules/estadisticas/components/citas/filtros.component.ts

```diff
@@ -130,7 +142,7 @@ export class FiltrosComponent implements AfterViewInit, OnChanges {
     }
 
     loadPrestaciones(event) {
-        this.servicioPrestacion.get({}).subscribe(result => {
+        this.servicioPrestacion.get({id: this.auth.getPermissions('dashboard:tipoPrestacion:?')}).subscribe(result => {
```

Comment:
Acá si  `this.auth.getPermissions('dashboard:tipoPrestacion:?')` devuelve `*`, no se envía ningún parámetro.